gesdinet/jwt-refresh-token-bundle/1.0/src/Entity/RefreshToken.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gesdinet\JWTRefreshTokenBundle\Entity\RefreshToken as BaseRefreshToken;
+
+/**
+ * @ORM\Entity
+ */
+class RefreshToken extends BaseRefreshToken
```

Comment:
When I use it like this, the bundle tries to use the table `refresh_token` without `s`. Is this table name change intended?
I've added `@ORM\Table("refresh_tokens")` to keep the current name.